tasks/php_apache.yml

```diff
@@ -1,5 +1,5 @@
   - name: LAMP | FAST-CGI configuration for php
-    template: src={{role_dir}}/templates/apache2/conf/fastcgi.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
+    template: src={{role_dir}}/templates/apache2/conf/fastcgi-{{ php_family_prefix }}.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
```

Comment:
An option could be avoid using that variable, but php_family_prefix=php still remains "php" as you specified in php_apt_switchversion.yml, right?